workflows/rnaseq/downstream/gene-patterns.Rmd

```diff
@@ -108,41 +110,71 @@ for (name in names(ll)){
         # col="another.column",
 
         # NOTE: see above note
-        minc=low.minc,
+        minc=minc,
 
         plot=FALSE
 
         )
+}
+```
+
+```{r plot_degpatterns, cache=TRUE, fig.width=12, results='asis', dependson='final_clusters'}
+# unique list of genes in clusters
+uniq.genes <- unique(unlist(lapply(dp.list, function(x) x[['genes']])))
+
+# data frame to hold cluster memberships
+# NOTE: here we assume that all genes are present in any 'res' object.
+#       - this should be replaced by the geneid -> symbol mapping df
+cluster.df <- data.frame(row.names = uniq.genes,
+                         genes = uniq.genes,
+                         symbol = res.list[[1]][['res']]$symbol[uniq.genes])
```

Comment:
I think it's a reasonable assumption that the first res has all genes. But should we hold off on merging this until that mapping df is implemented?

Reply:
yes, agreed. this is somewhat of a placeholder